### ISSUE TITLE
After Edge TPU conversion, model predicts always the same value

### ISSUE THREAD

**Issue by stefano555** [google-coral/edgetpu#295]
Everything can be found in a shared google document at this link https://drive.google.com/drive/folders/1NmMyL1w2FcQJcbmfeHzCIz-TFbiF6n6B?usp=sharing

In the file "quick_tf_lite_test.ipynb" I created a simple model to run inference on an MNIST dataset. Validation accuracy was 98% and the model can be found in the folder under the name "test_addition.pb". After that, I converted the model to a tf lite full integer quantisation optimisation uint8, with efficiency 100%. Model can found under the name "test_addition.tflite".

I subsequently converted the tflite model using the edge tpu compiler. Model and log in the folder. 
Output of the compiler was:

Edge TPU Compiler version 15.0.340273435

Model compiled successfully in 173 ms.

Input model: test_addition.tflite
Input size: 1.15MiB
Output model: test_addition_edgetpu.tflite
Output size: 1.22MiB
On-chip memory used for caching model parameters: 31.50KiB
On-chip memory remaining for caching model parameters: 7.81MiB
Off-chip memory used for streaming uncached model parameters: 1.13MiB
Number of Edge TPU subgraphs: 1
Total number of operations: 9
Operation log: test_addition_edgetpu.log
See the operation log file for individual operation details.

Edge TPU Compiler version 15.0.340273435

I then tested the model with a Coral Edge TPU (the usb version) using the code in the file "edge_test.ipynb" on a Ubuntu machine. The results were that the edge tpu predicted all the time the same value, 7.

I have used the TensorFlow lite API.

**Comment by manoj7410** [comment]
@stefano555 Were you able to fix this issue ?

**Comment by manoj7410** [comment]
Feel free to reopen if the issue still persists.
